extension/src/prototypes/ui-components/LoadingAnimationIcon.tsx

```diff
@@ -0,0 +1,49 @@
+import { keyframes, styled } from "@mui/material";
```

Comment:
This directory is for prototypes, so I think you should add these ui-components in [shared/ui-components](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/tree/main/extension/src/shared/ui-components)

Reply:
Updated, thank you!